scenes/root/Root.gd

```diff
@@ -71,6 +73,20 @@ func register_items():
 		"PlateLegs", "res://scenes/items/equipment/armour/platelegs/PlateLegs.tscn"
 	)
 
+func parse_cmd_arguments():
+	var args:PackedStringArray = OS.get_cmdline_args()
+	if not args.is_empty():
+		J.logger.info("Found launch arguments. ", str(args))
+		
+	for arg in args:
+		if arg == "j_client":
+			$SelectRunMode/VBoxContainer/RunAsClientButton.pressed.emit()
+			break
+			
+		elif "j_server" in args:
```

Comment:
Isn't this weird to do the "in" check of "args" while looping over "args"?

I think it's best to pick one mechanism.  

Option 1:
```
for arg in args:
    if arg == "j_client":
        start client
        return
    elif arg == "j_server":
        start server
        return
```

Option 2:
```
if "j_client" in args:
   start client
    return
elif "j_server" in args:
    start server
    return
```

Reply:
Yep, it is a remainder from an earlier failed approach.  
Will fix later today.